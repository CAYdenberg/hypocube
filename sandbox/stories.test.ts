import examples from './stories';
import renderer from 'react-test-renderer';

examples.forEach(eg => {
  test(`Test: ${eg.name}`, () => {
    const output = renderer.create(eg.render({ isCanvas: false }));
    expect(output).toMatchSnapshot();
  });
});
