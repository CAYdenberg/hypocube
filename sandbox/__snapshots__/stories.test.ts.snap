// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test: B.C. COVID-19 Vaccinations - Bar 1`] = `
<div
  style={
    Object {
      "height": 300,
      "maxWidth": 300,
      "minWidth": "100%",
      "position": "relative",
    }
  }
>
  <div>
    <svg
      height={300}
      width={300}
    >
      <g>
        <path
          d="M50,250L280,250"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <path
          d="M88.33333333333334,250L88.33333333333334,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={88.33333333333334}
          y={278}
        >
          2021/01/17
        </text>
        <path
          d="M126.66666666666666,250L126.66666666666666,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={126.66666666666666}
          y={278}
        >
          2021/01/18
        </text>
        <path
          d="M165,250L165,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={165}
          y={278}
        >
          2021/01/19
        </text>
        <path
          d="M203.33333333333331,250L203.33333333333331,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={203.33333333333331}
          y={278}
        >
          2021/01/18
        </text>
        <path
          d="M241.66666666666669,250L241.66666666666669,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={241.66666666666669}
          y={278}
        >
          2021/01/20
        </text>
        <path
          d="M280,250L280,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={280}
          y={278}
        >
          2021/01/21
        </text>
      </g>
      <g>
        <path
          d="M50,250L50,20"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
      </g>
      <path
        d="M83.33333333333334,152.60777777777778L93.33333333333334,152.60777777777778L93.33333333333334,250L83.33333333333334,250"
        fill="rgb(177, 0, 0)"
        stroke="#000"
        strokeWidth={1}
      />
      <path
        d="M121.66666666666666,152.63333333333333L131.66666666666666,152.63333333333333L131.66666666666666,250L121.66666666666666,250"
        fill="rgb(177, 0, 0)"
        stroke="#000"
        strokeWidth={1}
      />
      <path
        d="M160,121.63444444444445L170,121.63444444444445L170,250L160,250"
        fill="rgb(177, 0, 0)"
        stroke="#000"
        strokeWidth={1}
      />
      <path
        d="M198.33333333333331,152.63333333333333L208.33333333333331,152.63333333333333L208.33333333333331,250L198.33333333333331,250"
        fill="rgb(177, 0, 0)"
        stroke="#000"
        strokeWidth={1}
      />
      <path
        d="M236.66666666666669,102.90222222222222L246.66666666666669,102.90222222222222L246.66666666666669,250L236.66666666666669,250"
        fill="rgb(177, 0, 0)"
        stroke="#000"
        strokeWidth={1}
      />
      <path
        d="M275,76.83555555555554L285,76.83555555555554L285,250L275,250"
        fill="rgb(177, 0, 0)"
        stroke="#000"
        strokeWidth={1}
      />
    </svg>
  </div>
</div>
`;

exports[`Test: Canada COVID-19 Cases 1`] = `
<div
  style={
    Object {
      "height": 300,
      "maxWidth": 300,
      "minWidth": "100%",
      "position": "relative",
    }
  }
>
  <div
    onPointerDown={[Function]}
    onPointerLeave={[Function]}
    onPointerMove={[Function]}
    onPointerOut={[Function]}
    onPointerUp={[Function]}
  >
    <svg
      height={300}
      width={300}
    >
      <g>
        <path
          d="M50,250L280,250"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <path
          d="M52.90404040404049,250L52.90404040404049,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={52.90404040404049}
          y={278}
        >
          15
        </text>
        <path
          d="M75.96212121212113,250L75.96212121212113,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={75.96212121212113}
          y={278}
        >
          16
        </text>
        <path
          d="M99.0202020202021,250L99.0202020202021,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={99.0202020202021}
          y={278}
        >
          17
        </text>
        <path
          d="M122.07828282828291,250L122.07828282828291,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={122.07828282828291}
          y={278}
        >
          18
        </text>
        <path
          d="M145.13636363636357,250L145.13636363636357,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={145.13636363636357}
          y={278}
        >
          19
        </text>
        <path
          d="M168.19444444444454,250L168.19444444444454,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={168.19444444444454}
          y={278}
        >
          20
        </text>
        <path
          d="M191.25252525252517,250L191.25252525252517,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={191.25252525252517}
          y={278}
        >
          21
        </text>
        <path
          d="M214.31060606060615,250L214.31060606060615,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={214.31060606060615}
          y={278}
        >
          22
        </text>
        <path
          d="M237.3686868686868,250L237.3686868686868,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={237.3686868686868}
          y={278}
        >
          23
        </text>
        <path
          d="M260.4267676767676,250L260.4267676767676,260"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={260.4267676767676}
          y={278}
        >
          24
        </text>
      </g>
      <path
        d="M-1100,249.77L-1076.9419191919192,250L-1053.8838383838383,249.77L-1030.8257575757575,250L-1007.7676767676769,250L-984.709595959596,249.54L-961.651515151515,250L-938.5934343434344,250L-915.5353535353536,250L-892.4772727272726,250L-869.4191919191919,249.77L-846.3611111111111,250L-823.3030303030301,249.54L-800.2449494949495,250L-777.1868686868686,250L-754.1287878787878,250L-731.070707070707,250L-708.0126262626263,250L-684.9545454545455,250L-661.8964646464647,250L-638.8383838383838,250L-615.780303030303,250L-592.7222222222224,249.77L-569.6641414141415,250L-546.6060606060605,250L-523.5479797979798,250L-500.489898989899,249.77L-477.4318181818182,250L-454.3737373737374,250L-431.31565656565664,249.77L-408.25757575757564,249.77L-385.19949494949486,250L-362.1414141414141,249.54L-339.0833333333334,249.77L-316.0252525252525,248.62L-292.9671717171716,249.08L-269.9090909090909,249.31L-246.85101010101008,249.31L-223.79292929292936,249.31L-200.73484848484856,249.08L-177.67676767676775,247.24L-154.6186868686868,248.85L-131.560606060606,247.7L-108.50252525252517,247.01L-85.44444444444454,249.54L-62.386363636363726,243.33L-39.32828282828292,247.93L-16.270202020201936,232.52L6.787878787878867,248.85L29.845959595959677,237.58L52.90404040404049,212.51L75.96212121212113,235.51L99.0202020202021,208.83L122.07828282828291,217.11L145.13636363636357,217.11L168.19444444444454,173.17999999999998L191.25252525252517,205.84L214.31060606060615,107.63L237.3686868686868,88.54000000000002L260.4267676767676,143.96999999999997L283.4848484848486,68.07L306.5429292929292,102.8L329.6010101010102,44.379999999999995L352.6590909090908,88.08000000000001L375.7171717171718,-7.140000000000022L398.7752525252525,-9.670000000000002L421.8333333333331,12.410000000000018L444.8914141414141,-146.52L467.94949494949503,-15.190000000000005L491.00757575757575,125.57L514.0656565656567,-388.94L537.1237373737373,64.38999999999999L560.1818181818182,-51.069999999999986L583.2398989898988,-41.869999999999976L606.2979797979799,-97.98999999999998L629.3560606060606,-73.15L652.4141414141412,-39.10999999999997L675.4722222222222,23.910000000000004L698.5303030303028,-67.63L721.5883838383838,-61.65L744.6464646464646,-20.01999999999999L767.7045454545454,-348L790.7626262626263,-211.14999999999998L813.820707070707,-104.66L836.878787878788,-43.70999999999997L859.9368686868686,-215.75L882.9949494949497,-151.12L906.0530303030301,-270.0300000000001L929.1111111111109,-126.27999999999997L952.1691919191919,-122.60000000000002L975.2272727272725,-83.95999999999998L998.2853535353536,-132.26L1021.343434343434,-114.09L1044.401515151515,-102.82L1067.4595959595963,-144.45000000000002L1090.5176767676767,-116.16000000000003L1113.5757575757577,-183.77999999999997L1136.6338383838383,-114.09L1159.6919191919192,-342.93999999999994L1182.75,-84.19000000000001L1205.8080808080808,-39.34L1228.8661616161614,-90.17000000000003L1251.9242424242423,-96.60999999999997L1274.982323232323,-88.79000000000002L1298.0404040404042,-36.120000000000005L1321.098484848485,-19.790000000000013L1344.1565656565658,-19.790000000000013L1367.2146464646464,-15.650000000000006L1390.272727272727,-14.270000000000007L1413.330808080808,-28.99000000000002L1436.3888888888891,-20.939999999999987L1459.4469696969695,-36.81000000000003L1482.5050505050503,-8.979999999999976L1505.5631313131314,1.830000000000009L1528.6212121212122,1.1399999999999828L1551.6792929292928,1.1399999999999828L1574.7373737373734,-18.410000000000007L1597.7954545454545,-27.150000000000013L1620.8535353535353,-26.919999999999995L1643.9116161616162,30.35000000000001L1666.9696969696972,17.010000000000023L1690.0277777777783,26.670000000000005L1713.0858585858584,43.22999999999999L1736.1439393939393,22.990000000000002L1759.2020202020205,35.40999999999998L1782.2601010101007,72.44L1805.3181818181815,66.45999999999998L1828.3762626262628,63.929999999999986L1851.4343434343436,95.43999999999998L1874.4924242424242,93.36999999999999L1897.550505050505,105.56L1920.608585858586,94.05999999999999L1943.666666666666,128.56L1966.7247474747473,88.31000000000002L1989.7828282828282,111.77L2012.8409090909095,143.74L2035.8989898989898,139.83L2058.9570707070707,149.03L2082.0151515151515,149.72L2105.0732323232323,146.96L2128.1313131313136,166.97L2151.1893939393944,167.43L2174.2474747474744,175.47999999999996L2197.3055555555557,157.08L2220.3636363636365,161.22L2243.4217171717173,149.49L2266.479797979798,146.96L2289.537878787879,177.31999999999996L2312.59595959596,171.79999999999998L2335.65404040404,169.73L2358.712121212121,176.39999999999998L2381.7702020202023,163.52L2404.828282828282,211.82L2427.8863636363635,192.73L2450.9444444444443,177.55L2474.0025252525256,103.49L2497.0606060606056,188.59L2520.1186868686864,206.06999999999996L2543.176767676767,168.35L2566.2348484848485,176.63L2589.292929292929,198.71L2612.35101010101,201.93L2635.409090909091,153.17L2658.467171717172,202.16L2681.5252525252527,178.47L2704.5833333333335,175.93999999999997L2727.6414141414143,174.55999999999997L2750.6994949494956,211.82L2773.757575757576,204.46L2796.8156565656564,103.72L2819.873737373737,165.82L2842.9318181818185,171.11L2865.9898989898993,146.26999999999998L2889.0479797979797,154.55L2912.1060606060605,177.31999999999996L2935.1641414141413,182.61L2958.2222222222217,72.89999999999999L2981.280303030303,126.94999999999999L3004.338383838384,177.09L3027.3964646464647,110.16L3050.454545454545,85.09000000000002L3073.5126262626263,168.35L3096.570707070707,176.63L3119.628787878787,93.13999999999999L3142.686868686868,158L3165.744949494949,138.22L3188.80303030303,176.39999999999998L3211.8611111111113,111.08L3234.9191919191926,194.34L3257.977272727273,193.65L3281.0353535353534,202.85L3304.0934343434346,92.21999999999998L3327.1515151515155,163.98L3350.209595959596,168.58L3373.2676767676767,131.32L3396.325757575758,193.65L3419.3838383838392,199.63L3442.4419191919187,92.21999999999998L3465.5,172.71999999999997L3488.5580808080804,177.78L3511.6161616161617,140.29L3534.674242424242,152.25L3557.732323232323,199.4L3580.7904040404037,208.83L3603.8484848484845,200.78L3626.9065656565654,50.82L3649.964646464646,175.47999999999996L3673.022727272727,200.09L3696.080808080808,90.38000000000001L3719.1388888888887,194.57L3742.19696969697,190.89L3765.2550505050513,71.28999999999999L3788.3131313131316,178.93000000000004L3811.371212121211,140.29L3834.4292929292933,145.12L3857.4873737373737,133.62L3880.5454545454545,181.69000000000003L3903.6035353535353,192.73L3926.661616161617,13.099999999999994L3949.7196969696965,134.08L3972.777777777778,130.63L3995.8358585858587,115.22000000000001L4018.893939393939,108.78L4041.952020202021,164.9L4065.0101010101007,162.83L4088.068181818182,156.85L4111.126262626263,-86.25999999999999L4134.184343434343,163.06L4157.242424242425,61.170000000000016L4180.300505050504,84.4L4203.358585858585,137.76L4226.416666666667,140.52L4249.474747474747,-85.33999999999999L4272.532828282829,65.53999999999999L4295.59090909091,31.04000000000001L4318.648989898989,13.79000000000002L4341.707070707071,-20.01999999999999L4364.765151515151,122.57999999999998L4387.823232323233,-12.890000000000008L4410.881313131313,-171.59L4433.939393939395,-42.56L4456.997474747474,0.679999999999982L4480.055555555556,-14.039999999999981L4503.113636363637,-124.89999999999998L4526.171717171716,7.810000000000015L4549.229797979799,-131.33999999999997L4572.287878787879,-161.92999999999998L4595.345959595959,-157.56L4618.404040404041,-236.68L4641.462121212121,-169.29L4664.520202020201,-238.51999999999998L4687.578282828284,-50.38000000000001L4710.636363636363,-242.43L4733.694444444444,-413.09000000000003L4756.752525252526,-295.56L4779.810606060605,-123.52000000000002L4802.868686868687,-290.5L4825.926767676767,-415.85L4848.984848484848,-242.66L4872.042929292929,-126.27999999999997L4895.101010101011,25.750000000000007L4918.15909090909,-695.5299999999999L4941.217171717171,-342.25000000000006L4964.275252525253,-278.54L4987.333333333332,-340.17999999999995L5010.391414141414,218.03L5033.449494949495,-609.51L5056.507575757574,-510.61L5079.565656565656,-264.74L5102.623737373737,-368.00999999999993L5125.681818181819,-343.63L5148.739898989898,-417.46000000000004L5171.79797979798,-102.36L5194.85606060606,-399.29L5217.914141414141,-692.77L5240.972222222223,-377.9L5264.030303030302,-362.48999999999995L5287.088383838385,-443.91L5310.146464646465,-567.88L5333.204545454546,-296.02L5356.262626262627,-287.28L5379.320707070707,-561.67L5402.378787878789,-839.7400000000001L5425.436868686869,-389.86L5448.494949494949,-390.31999999999994L5471.553030303031,-990.39L5494.611111111111,-749.58L5517.669191919193,-657.12L5540.727272727273,-870.3300000000002L5563.785353535353,-772.3500000000001L5586.843434343435,-666.3199999999999L5609.901515151516,-1009.02L5632.959595959596,-847.0999999999999L5656.017676767677,-768.8999999999999L5679.075757575758,-739.23L5702.133838383838,-1137.59L5725.19191919192,-776.2599999999999L5748.25,-789.14L5771.30808080808,-852.6199999999999L5794.366161616162,-897.01L5817.424242424242,-890.11L5840.482323232322,-841.35L5863.540404040405,-1312.85L5886.598484848484,-964.86L5909.656565656564,-890.34L5932.714646464647,-1056.17L5955.772727272727,-1119.42L5978.830808080809,-1055.25L6001.888888888889,-988.09L6024.946969696969,-1565.85L6048.005050505051,-1013.85L6071.063131313131,-1234.42L6094.121212121211,-1190.72L6117.179292929292,-1226.37L6140.237373737373,-1211.42L6163.295454545454,-1200.15L6186.353535353535,-1526.52L6209.411616161617,-1169.33L6232.469696969697,-1221.77L6255.527777777777,-1287.78L6278.585858585858,-1294.45L6301.643939393939,-1126.32L6324.70202020202,-1110.68L6347.760101010102,-1676.7100000000003L6370.818181818182,-1186.12L6393.876262626263,-1247.76L6416.934343434344,-1361.38L6439.992424242426,-1291L6463.050505050505,-1235.57L6486.108585858586,-1131.6100000000001L6509.166666666667,-1492.71L6532.224747474747,-1189.8L6555.282828282829,-1341.83L6578.34090909091,-1287.32L6601.398989898991,-152.27000000000004L6624.457070707071,-466.22L6647.515151515152,-2073L6670.573232323232,-1236.95L6693.631313131313,-1633.47L6716.689393939395,-1695.57L6739.747474747474,-1392.89L6762.805555555557,22.07L6785.863636363636,-793.05L6808.921717171716,-3462.4299999999994L6831.979797979799,-2063.34L6855.03787878788,-1417.5L6878.095959595959,-1608.8600000000001L6901.154040404041,-1642.21L6924.212121212121,-1874.51L6947.270202020201,-1680.3900000000003L6970.328282828283,-1425.09L6993.386363636364,-1664.7499999999998L7016.4444444444425,-1162.43L7039.502525252525,-1391.97L7062.560606060605,-1462.81L7085.618686868687,-1332.4L7108.676767676768,-1212.57L7131.734848484848,-1195.32L7154.79292929293,-1226.6L7177.851010101009,-906.4399999999999L7200.90909090909,-1061.23L7223.967171717171,-1155.53L7247.025252525253,-1090.21L7270.083333333333,-921.6200000000001L7293.141414141413,-826.1700000000001L7316.199494949496,-1041.91L7339.257575757574,-679.4300000000001L7362.315656565656,-743.6000000000001L7385.373737373737,-855.38L7408.431818181817,-860.21L7431.4898989898975,-695.3000000000001L7454.54797979798,-654.59L7477.606060606062,-785.46L7500.664141414142,-428.04L7523.722222222223,-501.40999999999997L7546.780303030304,-665.86L7569.838383838383,-701.28L7592.896464646465,-500.72L7615.954545454546,-487.61L7639.012626262625,-624.69L7662.070707070708,-367.0899999999999L7685.12878787879,-501.64L7708.18686868687,-480.71L7731.244949494949,-366.16999999999996L7754.303030303031,-623.31L7777.361111111111,-256.9200000000001L7800.419191919191,-155.48999999999998L7823.477272727272,-720.3700000000001L7846.535353535354,-339.03L7869.5934343434365,-540.05L7892.651515151516,-459.09L7915.709595959596,-361.11L7938.7676767676785,-303.61L7961.825757575757,-544.19L7984.883838383837,-488.99L8007.94191919192,-427.34999999999997L8031,-473.81"
        fill="transparent"
        stroke="#5477a1"
        strokeWidth={1}
      />
    </svg>
  </div>
</div>
`;

exports[`Test: Click handler 1`] = `
Array [
  <p>
    0
  </p>,
  <div
    style={
      Object {
        "height": 300,
        "maxWidth": 300,
        "minWidth": "100%",
        "position": "relative",
      }
    }
  >
    <div>
      <svg
        height={300}
        width={300}
      >
        <g
          onPointerDown={[Function]}
        >
          <path
            d="M27.272727272727273,272.7272727272727L95.45454545454545,68.18181818181819L163.63636363636363,272.7272727272727"
            fill="#000"
            stroke="#000"
            strokeWidth={1}
          />
        </g>
      </svg>
    </div>
  </div>,
]
`;

exports[`Test: Filled line 1`] = `
<div
  style={
    Object {
      "height": 300,
      "maxWidth": 300,
      "minWidth": "100%",
      "position": "relative",
    }
  }
>
  <div>
    <svg
      height={300}
      width={300}
    >
      <path
        d="M27.272727272727273,272.7272727272727L95.45454545454545,68.18181818181819L163.63636363636363,272.7272727272727"
        fill="#000"
        stroke="#000"
        strokeWidth={1}
      />
    </svg>
  </div>
</div>
`;

exports[`Test: Line 1`] = `
<div
  style={
    Object {
      "height": 300,
      "maxWidth": 300,
      "minWidth": "100%",
      "position": "relative",
    }
  }
>
  <div>
    <svg
      height={300}
      width={300}
    >
      <path
        d="M27.272727272727273,272.7272727272727L95.45454545454545,68.18181818181819L163.63636363636363,272.7272727272727"
        fill="transparent"
        stroke="#000"
        strokeWidth={1}
      />
    </svg>
  </div>
</div>
`;

exports[`Test: Scatter plot 1`] = `
<div
  style={
    Object {
      "height": 300,
      "maxWidth": 300,
      "minWidth": "100%",
      "position": "relative",
    }
  }
>
  <div>
    <svg
      height={300}
      width={300}
    >
      <g>
        <path
          d="M50,270L290,270"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <path
          d="M50,270L50,280"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={50}
          y={298}
        >
          0
        </text>
        <path
          d="M170,270L170,280"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={170}
          y={298}
        >
          0.5
        </text>
        <path
          d="M290,270L290,280"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
        <text
          fill="#666"
          fontSize={16}
          style={
            Object {
              "fontFamily": "Helvetica, sans-serif",
            }
          }
          textAnchor="middle"
          x={290}
          y={298}
        >
          1
        </text>
      </g>
      <g>
        <path
          d="M50,270L50,10"
          fill="transparent"
          stroke="#666"
          strokeWidth={2}
        />
      </g>
      <g
        transform="translate(50, 270)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(74, 245.04000000000002)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(98, 223.20000000000002)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(122, 188.88)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(146, 173.28)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(170, 145.20000000000002)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(194, 124.4)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(218, 103.60000000000001)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(242, 82.80000000000001)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <g
        transform="translate(266, 61.999999999999986)"
      >
        <path
          d="M4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,-4.51351666838205,0A4.51351666838205,4.51351666838205,0,1,1,4.51351666838205,0"
          fill="#5477a1"
          stroke="#5477a1"
          strokeWidth={1}
        />
      </g>
      <path
        d="M50,270C58,260.9578183910881,66,251.91563678217616,74,245.04000000000002C82,238.16436321782388,89.99999999999999,233.4552712623834,98,223.20000000000002C106.00000000000001,212.94472873761663,114,197.1432781682904,122,188.88C130,180.61672183170958,138,179.89161606445518,146,173.28C154,166.66838393554482,162,154.1702575738891,170,145.20000000000002C178,136.22974242611093,186,130.78735363998848,194,124.4C202,118.01264636001153,210,110.68032786615697,218,103.60000000000001C226,96.51967213384305,233.99999999999997,89.69133489538373,242,82.80000000000001C250.00000000000003,75.90866510461629,258,68.95433255230813,266,61.999999999999986"
        fill="transparent"
        stroke="#5477a1"
        strokeWidth={1}
      />
    </svg>
  </div>
</div>
`;
